.github/workflows/buildAndTest.yml

```diff
@@ -125,6 +126,10 @@ jobs:
             -DLLVM_ENABLE_ASSERTIONS=ON
           cmake --build . --target install -- -j$(nproc)
 
+      # --------
+      # Build and test CIRCT in both debug and release mode.
```

Comment:
Why the duplicative comments?

Reply:
The big comment is supposed to be a section header and the comments before each step describe what that specific step does. The step comment says only "debug", the next step is "release".